src/slash_commands/pasta/ksiaze.txt

```diff
@@ -0,0 +1,13 @@
+Następną planetę zajmował czombiarz. Te odwiedziny trwały bardzo krótko, pogrążyły jednak Małego Księcia w głębokim smutku.
+- Co ty tu robisz? - spytał czombiarza, którego zastał siedzącego w milczeniu przed stołem i baterią wałówki.
+- Dupam - odpowiedział ponuro czombiarz.
+- Co to znaczy "dupam"? - spytał Mały Książę.
+- To znaczy, że spędzam swe życie przed madżongiem, stresując się każdym wyborem i narzekając na Discordzie po przegranych - odpowiedział zamyślony czombiarz.
+- A dlaczego dupasz? - pytał dalej Mały Książe.
+- Aby zapomnieć - odpowiedział czombiarz.
+- O czym zapomnieć? - zaniepokoił się Mały Książę, który już zaczął mu współczuć.
+- Aby zapomnieć, że jestem społecznie upośledzony - stwierdził czombiarz, schylając głowę.
+- Dlaczego jesteś społecznie upośledzony? - dopytywał się Mały Książę, chcąc mu pomóc.
+- Jestem społecznie upośledzony, bo dupam - zakończył czombiarz rozmowę i pogrążył się w milczeniu.
```

Comment:
we need to escape the dashes or else it will render as a Markdown list

Reply:
![](https://media.tenor.com/5MeqrF1RgM4AAAAC/faktycznie-ok.gif)

Reply:
ugh

Reply:
dorime https://github.com/riichi/chombot/pull/176

Reply:
interimo adapare dorime